util.go

```diff
@@ -47,6 +48,18 @@ func parallelism() uint32 {
 	return numCores
 }
 
+// inthash64 is a hash function based on murmurhash3 64-bit finalizer.
```

Comment:
Probably the comment is outdated =)

Reply:
True :)

Thanks for the review! Will fix the comment.